h5c/vic/src/vic-webapp/e2e/vch-create-wizard/1-basic.e2e-spec.ts

```diff
@@ -195,29 +196,64 @@ describe('VCH Create Wizard - Basic', () => {
-
-    browser.driver.getCapabilities().then(caps => {
-      browser.browserName = caps.get('browserName');
-      if (browser.browserName.toLowerCase() === 'chrome') {
-        browser.actions().mouseMove(allVicActions).click().perform();
-      } else {
-        page.clickByText('#applicationMenuContainer .k-item .k-link', 'All VIC Actions');
-      }
+  if (browser.params.hostAffinity === 'true') {
+    it('should redirect to VCH VM and select Cluster from VCH related objects', () => {
+      page.navigateToVchVm(namePrefix + specRunId);
+      browser.switchTo().defaultContent();
+      browser.sleep(defaultTimeout);
+      const vch = new RegExp(namePrefix + specRunId);
+      page.waitForElementToBePresent('a[title=Cluster]', 'css');
+      page.clickByCSS('a[title=Cluster]');
+      });
+
+    it('should navigate to clusters actions and settings', () => {
+      browser.switchTo().defaultContent();
+      browser.sleep(defaultTimeout);
+      page.waitForElementToBePresent('a.summary-action-link', 'css');
+      page.clickByCSS('a.summary-action-link');
+      browser.sleep(defaultTimeout);
+      const clustersActions = element(by.cssContainingText('#applicationMenuContainer .k-item .k-link', 'Settings'));
+      browser.driver.getCapabilities().then(caps => {
+        browser.browserName = caps.get('browserName');
+        if (browser.browserName.toLowerCase() === 'chrome') {
+          browser.actions().mouseMove(clustersActions).click().perform();
+        } else {
+          page.clickByText('#applicationMenuContainer .k-item .k-link', 'Settings');
+        }
+      });
     });
 
-    browser.sleep(defaultTimeout);
-    expect(browser.isElementPresent(by.cssContainingText('.vui-menuitem-label-text', 'New Virtual Container Host...'))).toBeTruthy();
-    expect(browser.isElementPresent(by.cssContainingText('.vui-menuitem-label-text', 'Delete Virtual Container Host'))).toBeTruthy();
-  });
+    it('should validate the additions of the vch to the host/virtual machines group', () => {
+      page.waitForElementToBePresent('//div[2]/ul/li[3]/a', 'xpath');
+      page.clickByXpath('//div[2]/ul/li[3]/a');
+      const vch = new RegExp(namePrefix + specRunId);
+      browser.switchTo().defaultContent();
+      expect(element(by.css('span[title=' + vch + ']')).isPresent).toBeTruthy();
+    });
+  } else {
+    it('should redirect to VCH VM and display Create Wizard menu items', () => {
+      page.navigateToVchVm(namePrefix + specRunId);
+      browser.switchTo().defaultContent();
+      browser.sleep(defaultTimeout);
+      page.waitForElementToBePresent('a.summary-action-link', 'css');
+      page.clickByCSS('a.summary-action-link');
+      // wait for menu items to be calculated
+      browser.sleep(defaultTimeout);
+      const allVicActions = element(by.cssContainingText('#applicationMenuContainer .k-item .k-link', 'All VIC Actions'));
```

Comment:
put '#applicationMenuContainer .k-item .k-link' in const since used multiple times